sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -74,7 +86,36 @@ internal class RadarApiClient(
         )
     }
 
-    internal fun getConfig() {
+    data class RadarMeta(
+        val config: JSONObject?,
+        val remoteTrackingOptions: RadarTrackingOptions?
+    )
+
+    private fun parseMeta(res: JSONObject?): RadarMeta {
```

Comment:
Then for these parsing methods, you can follow the existing conventions we have for other model classes